code/headers/lidar.hpp

```diff
@@ -0,0 +1,110 @@
+#include <lidar_packets.hpp>
+#include <usart_connection.hpp>
+
+namespace r2d2 {
+    namespace measuring_distance {
```

Comment:
Please change the namespace to `distance`, this is a bit ugly and out of line with the other modules